.changeset/ten-plants-kneel.md

```diff
@@ -0,0 +1,6 @@
+---
+'@crowdstrike/glide-core': minor
+---
+
+- Menu Button and Menu Link have been combined and renamed to Option.
```

Comment:
"Option" because it's usable in Menu and Split Button Secondary Button. And it'll also be usable in Select.